### ISSUE TITLE
Retrospect 5.6.0 : no correct subtitles for kijk.nl

### ISSUE THREAD

**Issue by FrankProg27** [retrospect-addon/plugin.video.retrospect#1705]
### Describe the bug

Until now I have been using retrospect very pleasantly for years and am using the latest version of Retrospect (5.6.0.) and Kodi 20.1) on a Raspberry PI4/4GB and a Sony KD-49XH9505 television (Android). Both have automatic updates enabled and are up-to-date.
A week ago, I noticed an strange phenomenon after enabling the subtitles for watching kijk.nl.. After enabling, Retrospect showed ( example and shortened ) HADGREBD-2023-05-09_0000001.jpg in the subtitle area instead of the actual subtitle. Looking at  broadcasts of the same program of the same channel (in this case SBS6, "de grote verbouwing Australia") of the week before, showed the correct subtitles. Another example is "Urk!" (only shows .jpg filename). NPO Start and RTL XL do not have this issue. 
This should be a bug, since it worked flawlessly a week ago.

## Expected Behavior

Show the correct subtitles for kijk.nl.

## Actual Behavior

Retrospect showed like HADGREBD-2023-05-09_0000001.jpg in the subtitle area instead of the actual subtitle.
NPO Start and RTL XL do not have this issue. 

## Possible Fix

Check if kijk.nl changed the way they provide their subtitles an implement a solution in Retrospect. Thank you for trying.

### To Reproduce

1. Open channel kijk.nl and look for "de grote verbouwing Australia", as broadcasted on 14-05-2023, this would show the issue (see picture 20230516_143142.jpg).
2. Open channel kijk.nl and look for "de grote verbouwing Australia", as broadcasted on 07-05-2023, this would not show the issue (see picture 20230516_143234.jpg).

### Debuglog

The **debug** retrospect.log can be found here:
[retrospect.log](https://github.com/retrospect-addon/plugin.video.retrospect/files/11487927/retrospect.log)

### Screenshots 

![20230516_143142](https://github.com/retrospect-addon/plugin.video.retrospect/assets/133771070/a2d663d3-13ab-478d-bc2d-933cbc0a784d)
![20230516_143234](https://github.com/retrospect-addon/plugin.video.retrospect/assets/133771070/ee8f27c5-abd5-471a-92a7-2983ec6ead03)

## Additional context or screenshots (if appropriate)

tried to accomplish looking back TV shows with the correct subtitles, tried to re-install Kodi [did not help] and tried to reinstall input stream adaptive[did not help.]

### Your Environment
Used Operating system:

 - [X] Android 
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [X] Raspberry-Pi
 - [ ] Windows
 - [ ] Windows UWP/

Used versions:
- Kodi version: 20.1
- Retrospect version: 5.6.0

**Comment by basrieter** [comment]
Do you have an example for RTL XL?

**Comment by basrieter** [comment]
Install this zip via the "install from zip" should fix Kijk.nl: 

https://github.com/retrospect-addon/plugin.video.retrospect/archive/refs/heads/bug-1705.zip

**Comment by FrankProg27** [comment]
> Do you have an example for RTL XL?

My apologies, phenomenon did not appear on the RTL XL channel, on that channel I first noticed blocky images after an update of inputstream adaptive. For the raspberry pi I was able to sucessfully use a previous version. I could achieve this for the Sony/android. This issue was visible on NPO, RTL XL and KIJK.NL. Will create a new issue for this and update the description of  issue 1705.
